app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
* Kindly ensure your devise setup Asks for confirmation of email during signup as stated in [the requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.) 
* Also when i try to reset my password i get this error `Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true`, Kindly fix this error [Following this devise setup instructions helped fix this error](https://github.com/heartcombo/devise#:~:text=config.action_mailer.default_url_options%20%3D%20%7B%20host%3A%20%27localhost%27%2C%20port%3A%203000%20%7D)
![image](https://user-images.githubusercontent.com/52497246/208011258-75506b78-21e7-4e92-a7f4-245833a0b2d4.png)